Myra-0.0.1/src/com/likya/myra/jef/jobs/JobHelper.java

```diff
@@ -148,32 +148,39 @@ public static ProcessBuilder parsJobCmdArgs(boolean isShell, String jobCommand,
 		ProcessBuilder processBuilder;
 
 		String realCommand = "";
-		String[] inlineArgs = null;
-
-		int indexOfSpace = jobCommand.indexOf(" ");
+//		String[] inlineArgs = null;
+//
+//		int indexOfSpace = jobCommand.indexOf(" ");
 
-		if (indexOfSpace > 0) {
-			realCommand = jobCommand.substring(0, indexOfSpace).trim();
-			inlineArgs = jobCommand.substring(jobCommand.indexOf(" ")).trim().split(" ");
-		} else {
-			realCommand = jobCommand.trim();
-		}
+//		if (indexOfSpace > 0) {
+//			realCommand = jobCommand.substring(0, indexOfSpace).trim();
+//			inlineArgs = jobCommand.substring(jobCommand.indexOf(" ")).trim().split(" ");
+//		} else {
+//			realCommand = jobCommand.trim();
+//		}
 
 		String[] commandArr;
-
-		if (isShell) {
-			commandArr = ValidPlatforms.getCommand(realCommand);
-		} else {
-			commandArr = new String[] { realCommand };
+		realCommand = jobCommand.trim();
+		
+		commandArr = ValidPlatforms.getCommand(realCommand, isShell);
+		
+		if (isShell && extArgValues != null && extArgValues.length() > 0) {
+			commandArr = concat(commandArr, extArgValues.trim().split(" "));
 		}
 
-		if (inlineArgs != null && inlineArgs.length > 0) {
-			commandArr = concat(commandArr, inlineArgs);
-		}
+//		if (isShell) {
+//			commandArr = ValidPlatforms.getCommand(realCommand);
+//		} else {
+//			commandArr = new String[] { realCommand };
+//		}
 
-		if (extArgValues != null && extArgValues.length() > 0) {
-			commandArr = concat(commandArr, extArgValues.trim().split(" "));
-		}
+//		if (inlineArgs != null && inlineArgs.length > 0) {
+//			commandArr = concat(commandArr, inlineArgs);
+//		}
+
+//		if (extArgValues != null && extArgValues.length() > 0) {
+//			commandArr = concat(commandArr, extArgValues.trim().split(" "));
+//		}
 
 		processBuilder = new ProcessBuilder(commandArr);
 
```

Comment:
@sahinkekevi JobHelper'da bir sürü comment vs var. Bunu anlamadım. Vaktin olduğunda bakabilir misin bir terslik olmasın.

Reply:
Terslik yok Taş'ım, kodun sahibine hürmetten silmedim :). Kritik yer olduğu için eski kodu belki tutmak istersin dedim, bazı kodlar commentli duruyor çünkü, istersen sen sil yada sen aldıktan sonra ben silerim.

Reply:
Estağ. Merge ettim. Biraderim, ne yaptığını anlamadım ama önemli de değil. Commentler kalabilir de silebilirsin de farketmez. Çalışıyorsa benim açımdan sorun yok :)